src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -2016,10 +2016,39 @@ components:
     AchievedQualificationResponse:
       title: AchievedQualificationResponse
       type: object
-      description: A qualification that a Prisoner has achieved, including the audit fields of who created/updated it.
+      description: A qualification that a Prisoner has achieved, including the audit fields of who created/updated it and when.
       allOf:
         - $ref: '#/components/schemas/AchievedQualification'
-        - $ref: '#/components/schemas/AuditedResource'
```

Comment:
We can no longer use `AuditedResource` for this type (`AchievedQualificationResponse`) because it contains the display name and prison ID fields, and these are the fields we cannot provide without a lot of rework.

Rather the extend `AuditedResource`, now we are simply defining the fields that we can provide